### ISSUE TITLE
MD029 is triggered on any multiline ordered list items

### ISSUE THREAD

**Issue by rroman81** [DavidAnson/vscode-markdownlint#6]
I am using the markdownlint v0.6.2 and I am seeing that the MD029 rule is always triggered even when using the sample provided inside the rule.
i.e.
```markdown
1. Do this.
2. Do that.
3. Done.
```
should pass the issue but it is generating green squiggle with MD029 tool tip.

**Comment by DavidAnson** [comment]
Duplicate of your similar issue https://github.com/DavidAnson/markdownlint/issues/45 against the underlying `markdownlint` library.
